arm/Microsoft.Compute/virtualMachines/.parameters/linux.min.parameters.json

```diff
@@ -18,8 +18,6 @@
         },
         "osDisk": {
             "value": {
-                "createOption": "fromImage",
-                "deleteOption": "Delete",
```

Comment:
Why remove this reference? Should the parameter file not try to use as many parameters as possible (even though they, in this case, use the default). 

Same comment for the others.

Reply:
As its the min file I was thinking it would make sense to show that there are default for it. The defaults are "smart" as in osDisk is default "fromImage" and dataDisks are "None".

Reply:
My bad, missed the `min` part